# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/thebes/gfxFontEntry.h
# Commit: a95e62dc9288
# Full Hash: a95e62dc9288041c95168c0158a7feaea822c851
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2022-08-16 16:01:29
# Regressor Bug: 1740530
# File Overlap Count: 1
# Description:
#   Bug 1740530 - patch 3 - Replace GetColorGlyphLayers and the layer-rendering code in gfxFont with a zero-copy implementation in COLRFonts. r=gfx-reviewers,lsalzman
#   
#   Depends on D152037
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D152038
# ==============================================================================

diff -r fdc71479fcd9 -r a95e62dc9288 gfx/thebes/gfxFontEntry.h
--- a/gfx/thebes/gfxFontEntry.h	Tue Aug 16 10:39:50 2022 +0000
+++ b/gfx/thebes/gfxFontEntry.h	Tue Aug 16 10:39:51 2022 +0000
@@ -58,9 +58,6 @@
 struct Face;
 struct Family;
 }  // namespace fontlist
-namespace gfx {
-struct DeviceColor;
-}
 }  // namespace mozilla
 
 typedef struct gr_face gr_face;
